common-graphql/src/main/scala/explore/common/ConstraintGroupQueriesGQL.scala

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.common
+
+import clue.GraphQLOperation
+import clue.annotation.GraphQL
+import explore.model
+import explore.model.ConstraintsSummary
+import explore.schemas.ObservationDB
+import java.time
+
+// gql: import explore.model.decoders._
+// gql: import explore.model.reusability._
+// gql: import io.circe.refined._
+// gql: import lucuma.ui.reusability._
+
+object ConstraintGroupQueriesGQL {
+
+  @GraphQL
+  trait ConstraintGroupObsQuery extends GraphQLOperation[ObservationDB] {
+    val document = """
+      query {
+        constraintSetGroup(programId: "p-2") {
+	        nodes {
+	          constraintSet {
+              name
+              cloudExtinction
+              imageQuality
+              skyBackground
+              waterVapor
+              elevationRange {
+                type: __typename
+                ... on AirMassRange {
+                  min
+                  max
+                }
+                ... on HourAngleRange {
+                  minHours
+                  maxHours
+                }
+              }
+            }
+	          observations {
+              nodes {
+                id
+              } 
+            }
+          }
+        }
+            
+	      observations(programId: "p-2") {
+          nodes {
+            id
+            observationTarget {
+              type: __typename
+              ... on Target {
+                targetId: id
+                targetName: name
+              }
+              ... on Asterism {
+                asterismId: id
+                asterismName: name
+              }
+            }
+            constraintSet {
+              imageQuality
+              cloudExtinction
+              skyBackground
+              waterVapor
+            }
+	        	status
```

Comment:
Yeah. It looks fine in my editor, but apparently there is a mix of spaces and tabs that didn't get normalized. Presumably because they are in a multiline quote.